src/Hedgehog/Tuple.fs

```diff
@@ -9,7 +9,7 @@ module private Tuple =
         x, f y
 
 
-module private GenTuple =
+module GenTuple =
```

Comment:
Did this change so tests can be written?

Reply:
Yep, see https://github.com/hedgehogqa/fsharp-hedgehog/pull/196#issuecomment-765893234.

Reply:
I'd be curious if `private` could be changed to `internal` to still avoid exposing this module for now.

Reply:
Or `internal` and combined with the [`InternalsVisibleTo`](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute) attribute.